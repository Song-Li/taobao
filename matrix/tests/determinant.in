SET A = sequence 1 1
SHOW A
COMPUTE determinant A
SET B = identity
SHOW B
SET C = matrix.add A B
SHOW C
COMPUTE determinant C

