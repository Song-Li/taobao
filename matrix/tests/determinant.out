> ok

> 1.00 2.00 3.00 4.00
5.00 6.00 7.00 8.00
9.00 10.00 11.00 12.00
13.00 14.00 15.00 16.00

> 0.00

> ok

> 1.00 0.00 0.00 0.00
0.00 1.00 0.00 0.00
0.00 0.00 1.00 0.00
0.00 0.00 0.00 1.00

> ok

> 2.00 2.00 3.00 4.00
5.00 7.00 7.00 8.00
9.00 10.00 12.00 12.00
13.00 14.00 15.00 17.00

> -45.00

> 
> 
bye
